src/intlGUIEditBox.cpp

```diff
@@ -1446,6 +1472,84 @@ void intlGUIEditBox::sendGuiEvent(EGUI_EVENT_TYPE type)
 	}
 }
 
+//! Create a vertical scrollbar
+void intlGUIEditBox::createVScrollBar()
+{
+	s32 fontHeight = 1;
+
+	if (OverrideFont) {
+		fontHeight = OverrideFont->getDimension(L"").Height;
+	} else {
+		IGUISkin* skin = Environment->getSkin();
+		if (skin) {
```

Comment:
please be more compact on the two conditions

if (IGUISkin* skin = Environment->getSkin()) {